package.json

```diff
@@ -1,44 +1,37 @@
 {
-  "name": "@dhis2/prop-types",
-  "version": "2.0.0",
-  "main": "./build/cjs/index.js",
-  "module": "./build/es/index.js",
-  "sideEffects": false,
-  "repository": "https://github.com/dhis2/prop-types.git",
-  "author": "Jan-Gerke Salomon <jgs.salomon@gmail.com>",
-  "license": "BSD-3-Clause",
-  "private": false,
-  "publishConfig": {
-    "access": "public"
-  },
-  "scripts": {
-    "test": "jest",
-    "docs": "jsdoc2md -t jsdoc2md/README.hbs src/*.js > README.md; echo",
-    "prebuild": "rm -rf ./build/** && yarn test",
-    "build:commonjs": "BABEL_ENV=commonjs babel src --out-dir ./build/cjs --copy-files --verbose",
-    "build:modules": "BABEL_ENV=modules babel src --out-dir ./build/es --copy-files --verbose",
-    "build": "NODE_ENV=production yarn build:commonjs && NODE_ENV=production yarn build:modules",
-    "lint:style": "d2-style js check --all"
-  },
-  "dependencies": {
-    "prop-types": "^15"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.8.4",
-    "@babel/preset-env": "^7.8.4",
-    "@babel/preset-react": "^7.8.3",
-    "@dhis2/cli-style": "6.0.0",
-    "babel-eslint": "^10.0.3",
-    "jest": "^25.1.0",
-    "jsdoc-to-markdown": "^5.0.3",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0"
-  },
-  "peerDependencies": {
-    "react": ">=15"
-  },
-  "files": [
-    "build"
-  ]
+    "name": "@dhis2/prop-types",
+    "version": "2.0.0",
+    "main": "./build/cjs/lib.js",
+    "module": "./build/es/lib.js",
+    "sideEffects": false,
+    "repository": "https://github.com/dhis2/prop-types.git",
+    "author": "Jan-Gerke Salomon <jgs.salomon@gmail.com>",
+    "license": "BSD-3-Clause",
+    "private": false,
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "test": "d2-app-scripts test",
+        "docs": "jsdoc2md -t jsdoc2md/README.hbs src/*.js > README.md; echo",
+        "build": "d2-app-scripts build",
+        "lint:style": "d2-style js check --all"
+    },
+    "dependencies": {
+        "prop-types": "^15"
+    },
+    "devDependencies": {
+        "@dhis2/cli-app-scripts": "^4.0.7",
+        "@dhis2/cli-style": "6.0.0",
+        "babel-eslint": "^10.0.3",
```

Comment:
I think this one is not needed anymore either, right? All 4 scripts ran successfully without this one on my machine

Reply:
No, should be fine with newer versions of cli-style.